packages/kit/src/core/adapt/prerender/prerender.js

```diff
@@ -167,8 +167,16 @@ export async function prerender({ cwd, out, log, config, build_data, fallback, a
 				if (location) {
 					mkdirp(dirname(file));
 
+					let url;
+					try {
+						url = new URL(location);
```

Comment:
This would fail in the case of non-absolute URLs (e.g. `location: /other`)

Reply:
Right! So it is either a string starting with a protocol `'http://'` or a string starting with `'/'` right? Or are there also other cases?

I wasn't sure about the error handling. What exactly should happen if `location` is an invalid url?
Right now it just uses the unencoded `location` instead

Reply:
Relative URLs (`./foo`) are also fine. This isn't the place to worry about whether they're valid or not — if they're not, then the next step (where we crawl `location`) will fail